vocabularies/HTML5.json

```diff
@@ -36,22 +39,18 @@
       "$DefaultValue": "_self",
       "@Common.Experimental": true,
       "@Core.Description": "Where to open a link for the annotated [URL](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.html#IsURL)",
-      "@Core.LongDescription": "Applicability of this term is governed by [`Validation.ApplicableTerms`](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Validation.V1.html#ApplicableTerms) annotations.\n          The value of this annotation is interpreted like the [target attribute](https://html.spec.whatwg.org/multipage/links.html#attr-hyperlink-target) in HTML."
+      "@Core.LongDescription": "Applicability of this term is governed by [`Validation.ApplicableTerms`](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Validation.V1.html#ApplicableTerms) annotations."
     },
     "LinkTargetType": {
       "$Kind": "TypeDefinition",
       "$UnderlyingType": "Edm.String",
+      "@Core.Description": "The values are interpreted like the [target attribute](https://html.spec.whatwg.org/multipage/links.html#attr-hyperlink-target) in HTML",
       "@Validation.AllowedValues": [
-        { "Value": "_self", "@Core.Description": "Open link in the current browsing context" },
-        { "Value": "_blank", "@Core.Description": "Open link in a new browsing context" },
-        { "Value": "_parent", "@Core.Description": "Open link in the parent browsing context" },
-        { "Value": "_top", "@Core.Description": "Open link in the top browsing context" }
```

Comment:
I liked these descriptions, the table of allowed values looks empty without them.